mapcache-el7/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM mapserver_rpm_el7:latest
+
+RUN yum install -y epel-release swig autoconf cmake httpd-devel libpng-devel libjpeg-devel \
+                   libjpeg-turbo-devel libcurl-devel libgeotiff-devel pcre-devel pixman-devel \
+                   fcgi-devel sqlite-devel libtiff-devel libdb-devel && yum -y clean all
+
+# Require mapserver support
+RUN yum install -y /mapserver*.rpm
+
+# Check out mapcache version 1.4
+RUN git clone git://github.com/mapserver/mapcache.git /root/mapcache
+WORKDIR "/root/mapcache"
+RUN git checkout c5b4683039501a511be7b4e0aafb5ad745d5dbca
```

Comment:
Tried using a couple of appropriate tags in the repo (1.4.0 and rel-1-4-1) but the code doesn't compile. The commit specified is the current HEAD in master.